frontend/challedu-app/src/App.css

```diff
@@ -87,3 +87,148 @@
     padding: 0px;
   }
 }
+
+/* Users admin table */
+
+* {
+  box-sizing: border-box;
+}
+
+.wrapper {
+  width: 100%;
+  max-width: 1000px;
+  margin: 1em auto;
+  padding: 1em;
+  font-size: .9rem;
+}
+.form-control {
+  font-size: .8rem!important;
+  padding: .375rem .5rem!important;
+}
+
+
+/* Table column sizing
+====================== */
+.id-cell {
+  width: 3%;
+}
+
+.fname-cell {
+  width: 20%;
+}
+
+.username-cell {
+  width: 15%;
+}
+
+.email-cell {
+  width: 27%;
+}
+
+.role-cell {
+  width: 15%;
+}
+
+.status-cell {
+  width: 19%;
+}
+
+/* Apply styles
+=============== */
+.FlexTable {
+  display: -webkit-box;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 0 3em 0;
+  padding: 0;
+  box-shadow: 0 0 40px rgba(0, 0, 0, 0.2);
+  background-color: rgba(255, 255, 255, 0.4);
+  
+}
+.FlexTable .FlexTable-row {
+  width: 100%;
+  display: -webkit-box;
+  display: flex;
+}
+.FlexTable .FlexTable-row .FlexTable-cell {
+  box-sizing: border-box;
```

Comment:
Why box-sizing: border-box again? Didn't we included it via the bootstrap file?

Reply:
Same here.